src/components/BaseCard.vue

```diff
@@ -40,13 +40,31 @@
 </template>
 
 <script>
+import get from 'lodash/get'
+
 export default {
   name: 'BaseCard',
   props: {
     post: {
       type: Object,
       default: () => {}
     }
+  },
+  computed: {
+    tagList() {
+      return get(this.$props.post, 'attributes.tags', []);
```

Comment:
You do not need to use `this.$props.post`.
you can access the post like this `this.post`